roles/ara_api/tasks/pre-requirements.yaml

```diff
@@ -27,12 +27,11 @@
 
 # Only attempt to elevate privileges if there are any missing packages
 - when: ara_api_missing_packages | length > 0
-  become: yes
+  become: true
   block:
-    # The apt module can not be used for this since it installs python-apt
```

Comment:
I haven't seen issues with installing python-apt so far. The only exception was docker images over a year old that have severely outdated caches

Reply:
I'd rather we keep this one -- for better or for worse I've personally experienced the issue when testing before release.

I've seen it happen outside of docker, notably in cloud images that may ship with outdated apt cache which prevents installing python-apt unless we first update the apt cache.

Reply:
Will do :-)